bzt/modules/tsung.py

```diff
@@ -405,8 +405,7 @@ def __gen_sessions(self, scenario):
             if request.body:
                 http_elem.set('contents', request.body)
 
-            headers = {}
-            headers.update(scenario.data.get('headers', {}))
+            headers = scenario.get_headers()
```

Comment:
Old code worked as deepcopy. Now update of headers will reflect in original dict.